RELEASE_PROCEDURE.md

```diff
@@ -24,68 +88,76 @@ will be working in an up-to-date local clone of the GitHub repository, where the
-1. Create a GitHub release, marked as a 'pre-release', incorporating
-   `CHANGELOG.md` information, and attaching the tarball and `sha256sums`.
-1. Notify the community about the RC via the Google Group and Slack.
+   `git tag -a -m "SingularityCE v3.11.0 Release Candidate 1" v3.11.0-rc.1`.
+1. Push the tag via `git push upstream v3.11.0-rc.1`.
+1. (Re-)trigger the CI workflow against the tag.
+1. Download the rpm and deb packages, which are generated as CI job artifacts,
+   from the CircleCI workflow for the tag.
+1. Create a tarball locally, via `git clean -fdx && mconfig -v && make dist`.
+1. Test intallation locally from the tarball.
+1. Compute the sha256sums of the tarball and packages, e.g. by collecting them
+   in a single directory and running `sha256sum * > sha256sums`.
+1. Create a GitHub pre-release, using the previous release as a guide and
+   incorporating `CHANGELOG.md` information. Make sure it is clear to users that
+   the release candidate is not for production use. Attach the tarball, rpms,
+   debs, and `sha256sums` to it.
+1. Notify the community about the release candidate via the Google Group and Slack.
 
 There will often be multiple release candidates issued prior to the final
-release of a new 3.Y.0 minor version.
-
-A small 3.Y.Z patch release may not require release candidates where the code
-changes are contained, confirmed by the person reporting the bug(s), and well
-covered by tests.
+release of a new major or minor version. Each new RC follows step 5 and above.
 
-## Creating a Final Release
+## Creating a Final Major / Minor Release
 
 1. Ensure the user and admin documentation is up-to-date for the new version,
-   branched, and tagged.
-   - [User Docs](https://www.sylabs.io/guides/latest/user-guide/) can be edited
+   has been branched, and tagged.
+   * [User Docs](https://www.sylabs.io/guides/latest/user-guide/) can be edited
      [here](https://github.com/sylabs/singularity-userdocs)
-   - [Admin Docs](https://www.sylabs.io/guides/latest/admin-guide/) can be edited
+   * [Admin Docs](https://www.sylabs.io/guides/latest/admin-guide/) can be edited
      [here](https://github.com/sylabs/singularity-admindocs)
 1. Ensure the user and admin documentation has been deployed to the sylabs.io
    website.
 1. Modify the `README.md`, `INSTALL.md`, `CHANGELOG.md` via PR against the
    release branch, so that they reflect the version to be released.
-1. Apply an annotated tag via `git tag -a -m "SingularityCE v3.8.0" v3.8.0`.
-1. Push the tag via `git push upstream v3.8.0`.
-1. Create a tarball via `mconfig -v && make dist`.
-1. Test intallation from the tarball.
-1. Compute the sha256sum of the tarball e.g. `sha256sum *.tar.gz > sha256sums`.
-1. Create a GitHub release, incorporating `CHANGELOG.md` information, and
-   attaching the tarball and `sha256sums`.
-1. Notify the community about the RC via the Google Group and Slack.
+1. Apply an annotated tag via `git tag -a -m "SingularityCE v3.12.0" v3.12.0`.
```

Comment:
Same here, worth signing?

```suggestion
1. Apply an annotated tag via `git tag -a -s -m "SingularityCE v3.12.0" v3.12.0`.
```

Reply:
Holding on this pending possible strategy for cross-siging / publishing keys.